src/Avalonia.Controls/Platform/IX11SpecificOptionsToplevelImplFeature.cs

```diff
@@ -0,0 +1,19 @@
+namespace Avalonia.Controls.Platform;
+
+public enum X11NetWmWindowType
+{
+    Normal,
+    Dialog,
+    Utility,
+    Menu,
+    Toolbar,
+    Splash,
+    Dock,
+    Desktop
+}
+
+public interface IX11SpecificOptionsToplevelImplFeature
```

Comment:
I checked in ILSpy, and this interface is fully public there in the ref assembly. I suppose our patch tool only looks for "Impl" in the end of the type name.
![image](https://github.com/AvaloniaUI/Avalonia/assets/3163374/783bfb52-0b37-4c01-b64c-fa48a9f0e9d5)